packages/fast-components-react-base/src/stack-panel/stack-panel.schema.ts

```diff
@@ -0,0 +1,58 @@
+export default {
+    $schema: "http://json-schema.org/schema#",
+    title: " stack panel",
```

Comment:
nit: space in from of title, would capitalize the "S".

Reply:
done